### PR TITLE
update malaria consortium pillow to use kafka

### PR DESCRIPTION
**Pull request by czue**
@benrudolph @snopoke first pillow to kafka!

this was way easier than I expected. ran locally and verified everything looks good. (it's still not leveraging the metadata for smarter filtering, but that's looking like it's going to be quite a bit of work to yank apart in fluff)

depends on https://github.com/dimagi/fluff/pull/113/ (already merged)

### CODE REVIEW

File: custom/reports/mc/models.py [hunk 1]
```diff
@@ -6,6 +8,7 @@
 from fluff.filters import NOTFilter, ANDFilter, ORFilter
 from fluff.models import SimpleCalculator
 from pillowtop.checkpoints.manager import get_default_django_checkpoint_for_legacy_pillow_class
+from pillowtop.pillow.interface import ConstructedPillow
```
Comment (dimagimon) [inline custom/reports/mc/models.py]: F401 'ConstructedPillow' imported but unused

**Comment by benrudolph** [comment]
:cloud: :cloud: :gun: 

looks good!

File: custom/reports/mc/models.py [hunk 2]
```diff
@@ -626,3 +628,9 @@ class Meta:
 
 
 MalariaConsortiumFluffPillow = MalariaConsortiumFluff.pillow()
+
+
+def get_pillow():
+    return MalariaConsortiumFluffPillow(
+        change_feed=KafkaChangeFeed(topic=topics.FORM, group_id='mc-fluff')
```
Comment (snopoke) [inline custom/reports/mc/models.py]: what does the `group_id` do?

Comment (czue) [inline custom/reports/mc/models.py]: it's the consumer group stuff from the presentation (lets you have multiple subscribers that get distributed messages from the same topic). we aren't using it for anything now

Comment (snopoke) [inline custom/reports/mc/models.py]: 👍

**Pull request merged**